web/app.js

```diff
@@ -617,6 +618,7 @@ let PDFViewerApplication = {
     this.findBar.reset();
     this.toolbar.reset();
     this.secondaryToolbar.reset();
+    this.contentDispositionFileName = null;
```

Comment:
To maintain better grouping between functionality here, let's move this line to just after https://github.com/mozilla/pdf.js/pull/8688/files#diff-529d1853ee1bba753a0fcb40ea778723R611 instead.